test/plugin/test_in_tail.rb

```diff
@@ -22,28 +24,42 @@ def teardown
   end
 
   def cleanup_directory(path)
+    unless Dir.exist?(path)
+      FileUtils.mkdir_p(path)
+      return
+    end
+
+    dir = File.join(Dir.tmpdir, SecureRandom.hex(10))
     begin
-      FileUtils.rm_f(path, secure: true)
+      files = Dir.glob("*", base: path)
```

Comment:
Could you add a comment as similar as the commit message?

Reply:
Fixed.